docs/SpatialEpigenome.rst

```diff
@@ -279,19 +269,14 @@ in specific regions of the tissue, visualize it's spatial distribution, and gain
 ------------------------------------------------------------------------
 
 **Add group coverages**
-Use the addGroupCoverages function to call on the input ArchR project object, proj_in_tissue, to add group coverages to the object. The groupBy 
-parameter specifies which metadata column to group the coverages by, in this case 'Clusters'.  
-This function calculates the average coverage of each peak in a single-cell RNA-seq dataset, grouped by 'Clusters'. The resulting object contains the 
-average coverage of each peak in each group, along with metadata about the peaks and the groups. This is done to obtain an understanding of the 
-coverage of peaks across different clusters to provide insights into various cluster characteristics.::
+
+It's important to understand how gene expression varies across different cell types or clusters. The addGroupCoverages function calculates the average coverage of each genomic region (or "peak") in the dataset, grouped by a specific metadata column (in this case, Clusters). By doing so, it allows us to compare the average expression of each peak across different cell types, providing insights into various cluster characteristics.::
```

Comment:
This function is more so for creating pseudo-bulk replicates to account for binary nature of single cell ATAC data and to prepare our data for peak-calling. In ATAC data, we dont have gene expression data however we do have gene score information which attempts to assigns an accessibility "score" to each gene according to the number of peaks near its TSS